java/src/jmri/jmrix/openlcb/swing/eventtable/EventTablePane.java

```diff
@@ -396,8 +426,6 @@ public Object getValueAt(int row, int col) {
 
         int lineIncrement = -1; // cache the line spacing for multi-line cells
 
-        static final String tagPrefix = "ID_OpenLCB_";  // Prefix of IdTag system name
```

Comment:
Can it cause any problems with changing the prefix? Are ID tags with this old prefix in use?

Reply:
It could, but I think it's unlikely that anybody unsuspecting is using it yet.  I haven't documented the use of the IdTag table with the Event Table anywhere (yet), as I was still working on that until now.